server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -317,6 +329,39 @@ public String executor() {
         }
     }
 
+    TransportResponse handleExtensionRequest(ExtensionRequest extensionRequest) throws Exception {
```

Comment:
The javadocs didn't move with the rest of the method.  As package private this may not be required per our settings but I think they bring value.